packages/block-editor/src/components/inner-blocks/index.native.js

```diff
@@ -97,7 +101,17 @@ function UncontrolledInnerBlocks( props ) {
 
 	const context = useBlockContext( clientId );
 
-	useNestedSettingsUpdate( clientId, allowedBlocks, templateLock );
+	useNestedSettingsUpdate(
+		clientId,
+		allowedBlocks,
+		prioritizedInserterBlocks,
+		__experimentalDefaultBlock,
+		__experimentalDirectInsert,
+		templateLock,
+		captureToolbars,
+		orientation,
+		layout
```

Comment:
Why is this PR adding all these other settings and not only the core feature `prioritizedInserterBlocks`?

Reply:
Because not doing that broke the Mobile implementation (cc @fluiddot).